### DISCUSSION TITLE
Label transfer from RNA to ATAC

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#681] [General]
Hi @rcorces @jgranja24 @archrdevs 

I have question posted here:

https://github.com/GreenleafLab/MPAL-Single-Cell-2019/issues/26#issue-852767112 

Thanks!

**Comment by rcorces** [comment]
Though I wasnt involved in this particular analysis, I assume "biological group" refers to CD34+ cells vs PBMCs etc.
So CD34 ATAC data was aligned to CD34 RNA data rather than doing all-to-all alignment where CD34+ cells could align to PBMCs for ex.
